website/content/docs/oss/configuration/worker/overview.mdx

```diff
@@ -0,0 +1,60 @@
+---
+layout: docs
+page_title: Worker - Overview
+description: |-
+  The worker stanza configures worker-specific parameters.
+---
+
+# `worker` Stanza
+
+The `worker` stanza configures Boundary worker-specific parameters. Boundary supports two different types of workers,
+differentiated by their means of authentication to Boundary.
+- [KMS Workers][] use a shared KMS to authenticate with controllers.
```

Comment:
```suggestion
- [KMS Workers][] use a shared KMS to authenticate with controllers
```
(The bullet point below doesn't have a period, so it's a mismatch)

Reply:
fixed 👍